src/main/java/gregtech/api/metatileentity/MetaTileEntity.java

```diff
@@ -1350,7 +1359,11 @@ public int getItemStackLimit(ItemStack stack) {
      * If placing an MTE with methods such as {@link World#setBlockState(BlockPos, IBlockState)},
      * this should be manually called immediately afterwards
      */
-    public void onPlacement() {}
+    public void onPlacement(EntityLivingBase placer) {
```

Comment:
This method taking in an EntityLivingBase should be a new method, and this parameterless method should just set the `owner` to null, so you don't have to explicitly pass in null like you do elsewhere in this PR.